### ISSUE TITLE
Mask2Former results City->ACDC

### ISSUE THREAD

**Issue by helen1c** [dingjiansw101/HGFormer#2]
Thank you, @dingjiansw101, for your awesome work!

In experiments with M2F, how long did you train M2F, and what was the batch size? I'm curious to reproduce your M2F results on the City->ACDC domain generalization. When I train M2F for 20,000 iterations on Cityscapes, I obtain a 25mIoU on ACDC, which is far from your results.

**Comment by helen1c** [comment]
And yes, when can we expect the code to be released?

**Comment by dingjiansw101** [comment]
> Thank you, @dingjiansw101, for your awesome work!
> 
> In experiments with M2F, how long did you train M2F, and what was the batch size? I'm curious to reproduce your M2F results on the City->ACDC domain generalization. When I train M2F for 20,000 iterations on Cityscapes, I obtain a 25mIoU on ACDC, which is far from your results.

I trained for 20k iterations, the batch size is 16.

**Comment by dingjiansw101** [comment]
> And yes, when can we expect the code to be released?

I just had time to clean the code recently.

**Comment by helen1c** [comment]
In the paper you said "we tested our models on ACDC". Which subset of ACDC? Test or val? If it is ACDC test, you evaluated every time on the evaluation benchmark?

Thanks! :)

**Comment by dingjiansw101** [comment]
> In the paper you said "we tested our models on ACDC". Which subset of ACDC? Test or val? If it is ACDC test, you evaluated every time on the evaluation benchmark?
> 
> Thanks! :)

Test set, we evaluated on the evaluation benchmark.

**Comment by helen1c** [comment]
Thank you. Regarding other datasets such as Mapillary, Synthia, GTA5, and BDD, from what I understand, there is only a validation subset available for each dataset (excluding GTA5).

For M, S, G, and B, what did you utilize as the test subset on which you presented the results in Table 2?

**Comment by dingjiansw101** [comment]
For M, S, G, and B, I used the validation set.
